R/mapping_using_bigQuery_drugExposure.R

```diff
@@ -0,0 +1,116 @@
+# Load necessary libraries
+library(dplyr)
+library(DBI)
+library(readr)

```

Comment:
A comment here would be good to briefly explain why the library(readr) package is being loaded